klayout/drc/README.md

```diff
@@ -2,44 +2,34 @@
 
 Explains how to use the runset.
 
-## Folder Structure
+## **Folder Structure**
```

Comment:
I don't think we need to make this bold (header are already emphased when rendered in github)

Reply:
Done